python/dgl/graphbolt/dataloader.py

```diff
@@ -75,7 +75,7 @@ def __iter__(self):
         yield from self.dataloader
 
 
-class DataLoader(torch_data.DataLoader):
+class DataLoader(MiniBatchTransformer):
```

Comment:
why not inherit from `IterDataPipe`?

Reply:
MiniBatchTransformer ensures returned objects are `gb.MiniBatch` objects.